planner/funcdep/extract_fd_test.go

```diff
@@ -213,6 +213,8 @@ func TestFDSet_ExtractFD(t *testing.T) {
 	is := testGetIS(t, tk.Session())
 	for i, tt := range tests {
 		comment := fmt.Sprintf("case:%v sql:%s", i, tt.sql)
+		require.NoError(t, tk.Session().PrepareTxnCtx(context.TODO()))
+		require.NoError(t, sessiontxn.GetTxnManager(tk.Session()).OnStmtStart(context.TODO()))
```

Comment:
They can both work for the current test. `tk.MustExec("begin")` will create a new explicit transaction and the current way creates a lazy transaction.   Because the current test will not build executor, so the tso will not be fetched for a lazy transaction.

Reply:
Both ways look good to me, and I'm not sure which one satisfies the developers who do not want to learn about txn manager. Let's keep it by now.